tests/test.js

```diff
@@ -108,3 +108,19 @@ describe('shim', function() {
 		delete Object.prototype[1];
 	});
 });
+
+describe('single function', function() {
+	var findAsFunction = function(func) {
+			return find.call(undefined, this, func, arguments[1]);
```

Comment:
Sorry, didn't get it. What should it do? I want to call `find` but pass params there and add current context as a first param.